block/sync.go

```diff
@@ -71,18 +68,16 @@ func (m *Manager) SettlementSyncLoop(ctx context.Context) error {
 
 				settlementBatch, err := m.SLClient.GetBatchAtHeight(m.State.NextHeight())
 				if err != nil {
-					return fmt.Errorf("retrieve batch: %w", err)
+					m.logger.Error("retrieve SL batch", "err", err)
+					break
```

Comment:
why do we break vs returning error?

Reply:
because if it temporary fails the access to SL, do you want to return error and quit the loop? this way it will try again for the next state update but without failing (the rollapp maybe operative and is just an issue with the hub node).

Reply:
thing is that I think that if we return an error it means the hub node retires exhausted.
in that case it could be due to:
1. bad rpc endpoitn (requires node operator inteference)
2. hub is down

either way I think both should raise node unhleathy and let the operator figure it out

Reply:
changed to error